FrontEnd/src/pages/AdminPage/MainPage/MainPage.jsx

```diff
@@ -5,6 +5,7 @@ function MainPage() {
     return (
         <div className={css['main-page-section']}>
             <h2>Вітаємо в панелі адміністратора.</h2>
+            <br />
```

Comment:
nit: it seems that you use `br` just for spacing. It's not the best practice. Consider using CSS for margins. https://stackoverflow.com/a/1726079

Reply:
Fixed.